update-completed-sprint-on-issue-closed/src/index.ts

```diff
@@ -0,0 +1,108 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+
+export default async function() {
+  try {
+    const token = core.getInput('repo-token');
```

Comment:
I don't see repo token defined anywhere as an input. Where does it come from?

Reply:
I don't remember exactly with callable workflows, but I think the `GITHUB_TOKEN` secret should be available.